lib/services/ngsi/ngsiService.js

```diff
@@ -44,6 +46,22 @@ var request = require('request'),
     queryMiddleware = [];
 
 /**
+ * It checks if the configuration file states the use of NGSIv3
+ *
+ * @param      {Object} newConfig            Configuration of the Context Server
```

Comment:
Missaligned?

Reply:
Fixed in d1bc5ac94a84def33800eb14ecbe95b18a7912bc